src/_media-queries.css

```diff
@@ -0,0 +1,3 @@
+@custom-media --breakpoint-not-small screen and (min-width: 30em);
```

Comment:
Technically, a tachyons update wouldn't even change these variables since we're overriding them outside of the tachyons node module. What I'm worried about is when we update tachyons modules that use breakpoints 

For example, if they depreciate the `not-small` breakpoint all of the instances in our markup that use that class will break, right?

```css
.fl { float: left;  _display: inline; }
.fr { float: right; _display: inline; }
.fn { float: none; }

@media (--breakpoint-not-small) {
  .fl-ns { float: left; _display: inline; }
  .fr-ns { float: right; _display: inline; }
  .fn-ns { float: none; }
}

@media (--breakpoint-medium) {
  .fl-m { float: left; _display: inline; }
  .fr-m { float: right; _display: inline; }
  .fn-m { float: none; }
}

@media (--breakpoint-large) {
  .fl-l { float: left; _display: inline; }
  .fr-l { float: right; _display: inline; }
  .fn-l { float: none; }
}
```